conf/global.config

```diff
@@ -1,5 +1,5 @@
 //MPRAflow version
-params.version="2.3.4"
+params.version="2.3.5"
```

Comment:
I think PR has multiple thinks (fixes and features) in it. So due to (semantic versioning)[https://semver.org/lang/de/] it is not a increase of the patch version.

Can you redirect the PR to the development branch? I think we have to do some steps (update documentation, fix some issues you found (hard coded BC length)) before creating a new release.